crates/rover-client/src/query/partial/fetch.rs

```diff
@@ -43,17 +44,23 @@ fn get_services_from_response_data(
     // get list of services
     let services = match service_data.implementing_services {
         Some(services) => Ok(services),
-        None => Err(RoverClientError::HandleResponse {
-            msg: "There are no implementing services in this graph".to_string(),
+        None => Err(RoverClientError::ExpectedFederatedGraph {
+            graph_name: service_name.to_string(),
```

Comment:
hmmmm - the mixing of graph and service here is quite confusing

Reply:
Oooh good catch. That's not even the right value!